### PR TITLE
Support setting specific variants unoverridable 

### PR DESCRIPTION
**Pull request by bratseth**

### CODE REVIEW

File: container-search/src/main/resources/configdefinitions/search.query.profile.config.query-profiles.def [hunk 1]
```diff
@@ -52,10 +52,14 @@ queryprofile[].queryprofilevariant[].inherit[] string
 queryprofile[].queryprofilevariant[].property[].name string
 # Content of profile variant
 queryprofile[].queryprofilevariant[].property[].value string
+# Whether the is overridable: "true", "false" or ""
```
Comment (arnej27959) [inline container-search/src/main/resources/configdefinitions/search.query.profile.config.query-profiles.def:55]: "whether the" what? (missing, i would guess "property") "is overridable"

File: container-search/src/main/resources/configdefinitions/search.query.profile.config.query-profiles.def [hunk 2]
```diff
@@ -52,10 +52,14 @@ queryprofile[].queryprofilevariant[].inherit[] string
...
+queryprofile[].queryprofilevariant[].property[].overridable string default=""
 # Content of profile variant
 queryprofile[].queryprofilevariant[].reference[].name string
 # Content of profile variant
 queryprofile[].queryprofilevariant[].reference[].value string
+# Whether the is overridable: "true", "false" or ""
```
Comment (arnej27959) [inline container-search/src/main/resources/configdefinitions/search.query.profile.config.query-profiles.def:61]: "whether the" what? (missing, i would guess "reference") "is overridable"

**Review by arnej27959 — APPROVED** [review]
query profiles keep getting more and more complicated...
but apart from one very minor comment this looks legit to me

File: container-search/src/main/resources/configdefinitions/search.query.profile.config.query-profiles.def (hunk 1 above)
Comment (bratseth) [reply container-search/src/main/resources/configdefinitions/search.query.profile.config.query-profiles.def]: Fixed, thanks.

**Comment by bratseth** [comment]
Yes, it would be nice to redo the whole thing, now that the full feature set is known, but I don't think I'll ever have the time.

**Pull request merged**